src/ui/button/button.tsx

```diff
@@ -4,6 +4,8 @@ import { FC, MouseEvent, ReactNode } from 'react'
 import cn, { Argument } from 'classnames'
 import { CSSProperties } from 'styled-components'
 
+import { LoaderWrapper } from '@components/loader'
```

Comment:
I think, that component from @ui should be isolated from other code, and shouldn't contains imports from other parts of code... May be we can move Loader and LoaderWrapper to @ui/

Reply:
good point